Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -169,23 +167,10 @@ public function setControllerContext(ControllerContext $controllerContext): void
      */
     public function getControllerContext(): ControllerContext
     {
-        if (isset($this->controllerContext)) {
```

Comment:
this logic was only introduced in 9.0 dev thats why we can so easily change it: https://github.com/neos/neos-development-collection/pull/4425